src/ArcanaPrime.sol

```diff
@@ -375,8 +397,8 @@ contract ArcanaPrime is ERC721AQueryable, ERC721ABurnable, Ownable, OperatorFilt
         return getBits(_getAux(_minter), 0, 2);
     }
 
-    function getPublicListMints(address _minter) public view returns (uint256) {
-        return getBits(_getAux(_minter), 2, 3);
+    function getTotalMints(address _minter) public view returns (uint256) {
+        return getBits(_getAux(_minter), 2, 5);
```

Comment:
Document this magic numbers somehow (can extract a well-named variable), explain why you chose those values in specific